eslint-plugin-expensify/prefer-str-method.js

```diff
@@ -0,0 +1,36 @@
+
+const _ = require('underscore');
+const lodashGet = require('lodash/get');
+
+const message = require('./CONST').MESSAGE.PREFER_STR_METHOD;
+const astUtil = require('./utils/astUtil');
+
+/**
+ * @param {Object} node
+ * @returns {Boolean}
+ */
+function isUsingIncorrectStrMethod(node) {
+    const strMethods = ['replaceAll'];
+    const callerName = lodashGet(node, 'callee.object.name');
+    if (!_.includes(strMethods, astUtil.getMethodName(node))) {
+        return;
```

Comment:
Should return false?

Reply:
ah yes technically it should since that is how we documented the value.